cmd/devstream/init.go

```diff
@@ -22,6 +22,11 @@ func initCMDFunc(cmd *cobra.Command, args []string) {
 	}
 
 	log.Info("Initialize started.")
+	if err := pluginmanager.CompareDtmMD5(); err != nil {
+		log.Error(err)
```

Comment:
"err" is generated by errorf(), which results in a sentence fragment, not a full sentence. However, log should be a full sentence. So, here it will not print pretty logs. We should do something like "Got error during xxx, err: %s" (for example, don't copy paste).

Reply:
Updated